backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -286,6 +286,40 @@ func TestCreatePipeline(t *testing.T) {
 	assert.Equal(t, pipelineExpected, pipeline)
 }
 
+func TestCreatePipeline_V2PipelineName(t *testing.T) {
+	tests := []struct {
+		name         string
+		namespace    string
+		pipelineName string
+	}{
+		{"v2-compat", "", "pipeline/v2-compat"},
```

Comment:
Is it more clear to specify the field like "name", "namespace" and "pipelineName" here?